src/lua/digest.lua

```diff
@@ -163,7 +209,7 @@ end
 local m = {
     base64_encode = function(bin, options)
         if type(bin) ~= 'string' or
-           options ~= nil and type(options) ~= 'table' then
+                options ~= nil and type(options) ~= 'table' then
```

Comment:
Unnecessary change, lets keep it as it is.

Reply:
it was a misprint. Fixed.

Reply:
I still see this change in the first commit.

Reply:
Sorry Vlad, my bad. Fixed. Typo in commit message is also fixed.